### ISSUE TITLE
I cannot use "dotdraw" function

### ISSUE THREAD

**Issue by anushkap** [casadi/casadi#950]
I am using JModelica to use CasADi. But I cannot use the function "dotdraw". I get "NameError" saying "name dotdraw is not definded". Is anybody knows why?

**Comment by Mathadon** [comment]
I just had the same issue. I needed to install ```pydot``` and then also the ```graphviz``` package on ubuntu.

**Comment by b4be1** [comment]
Is `dotdraw` included in CasADi 3.0.1? I am using `casadi-py35-np1.9.1-v3.0.1` build for MacOS. Apparently symbol `dotdraw` cannot be imported. The following
```python
from casadi.tools import dotdraw
```
returns

```
ImportError: cannot import name 'dotdraw'
```

**Comment by jaeandersson** [comment]
@b4be1 This is a closed issue. Please move this to a new, separate issue.
